test/certs/mkcert.sh

```diff
@@ -233,12 +233,14 @@ geneealt() {
 genee() {
     local OPTIND=1
     local purpose=serverAuth
+    local ku=
 
-    while getopts p: o
+    while getopts p:k: o
     do
         case $o in
         p) purpose="$OPTARG";;
-        *) echo "Usage: $0 genee [-p EKU] cn keyname certname cakeyname cacertname" >&2
+        k) ku="keyUsage = $OPTARG";;
```

Comment:
for consistency with the other options, I suggest 
```
ku=$OPTARG;;
```

Reply:
Need to check whether this works. For "purpose" there is a default value set, so the EKU is set in any case. As of now, there is no default for the KU, therefore this different way of handling it.

Reply:
Ah, I was not aware of this. 
So it the change would take more effort, you can just leave it as it is now.